bubble_sort.rb

```diff
@@ -0,0 +1,16 @@
+def bubble_sort(fig)
+  num = fig.length
+  loop do
+    sorted = false
+    (num - 1).times do |a|
+      if fig[a] > fig[a + 1]
+        fig[a], fig[a + 1] = fig[a + 1], fig[a]
+        sorted = true
+      end
+    end
+    break if sorted
+  end
+  fig
+end
+val = [4, 3, 78, 2, 0, 2]
+print bubble_sort val
```

Comment:
- This method returns [3, 4, 2, 0, 2, 78] instead of [0, 2, 2, 3, 4, 78].

Reply:
Thank you for the review. Will make necessary changes.